python/az/aro/azext_aro/custom.py

```diff
@@ -155,6 +158,93 @@ def aro_create(cmd,  # pylint: disable=too-many-locals
                        parameters=oc)
 
 
+def aro_validate(cmd,  # pylint: disable=too-many-locals
+                 client,  # pylint: disable=unused-argument
+                 resource_group_name,
+                 master_subnet,  # pylint: disable=unused-argument
+                 worker_subnet,  # pylint: disable=unused-argument
+                 vnet,  # pylint: disable=unused-argument
+                 location,  # pylint: disable=unused-argument
+                 client_id,  # pylint: disable=unused-argument
+                 client_secret,  # pylint: disable=unused-argument
+                 vnet_resource_group_name=None,  # pylint: disable=unused-argument
+                 pod_cidr=None,  # pylint: disable=unused-argument
+                 service_cidr=None,  # pylint: disable=unused-argument
+                 master_vm_size=None,  # pylint: disable=unused-argument
+                 disk_encryption_set=None,  # pylint: disable=unused-argument
+                 worker_vm_size=None,  # pylint: disable=unused-argument
+                 worker_vm_disk_size_gb=None,  # pylint: disable=unused-argument
+                 worker_count=None,  # pylint: disable=unused-argument
+                 ):
```

Comment:
This function call is missing the new features, apiserver_visibility, ingress_visibility, and version. For validating version you will need to [fetch the list from the RP](https://github.com/Azure/ARO-RP/blob/cab5146c98df281050a99c065bbb2a46af6ff8ce/python/az/aro/azext_aro/custom.py#L318) and validate dynamically.

Reply:
Added